packages/gcf-utils/README.md

```diff
@@ -4,6 +4,22 @@
 
 ## Usage
 
+### GCFLogger
+
+`GCFLogger` is a standardized logger for Google Cloud Functions. Note: this logger is a singleton shared across the 
+
+```
+import {GCFLogger, getLogger} from 'gcf-utils';
+
+let logger: GCFLogger = getLogger();
```

Comment:
```suggestion
const logger = getLogger();
```

You shouldn't have to declare the type as the compiler can figure it out from the assignment.

Reply:
sure, I can fix that. On second thought, since we're now not in a class anymore, any point having a `getLogger()` method or can I simply just export a logger const?

Reply:
I was having the exact same thought